recipes/phasecalculator/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "phasecalculator" %}
+{% set version = "1.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.developers.cam.ac.uk/ch/hunter/ssiptools/ssimpleapps/phasecalculator/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a513e670b5a194c68601c54e3530533cab752f18ed5f858eeb6a4138eec7a427
+
+build:
+  noarch: python
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+    - setuptools_scm
+  run:
+    - python >=3.6
+    - rdkit
+    - matplotlib
```

Comment:
Do you want to try matplotlib-base instead? If you don't need a backend, for example when your library will be used mostly on jupyternotebooks, that will make the installation lighter b/c you avoid downloading qt.

Reply:
It's mainly used through a CLI, so I think I need the matplotlib rather than base. (If I understand the build correctly).

Reply:
If you need a Window to pop-up you are correct. You'll need a GUI.